src/roles.json

```diff
@@ -1190,7 +1190,7 @@
     "team": "townsfolk",
     "firstNight": 29,
     "firstNightReminder": "Show the Pixie 1 in-play Townsfolk character token.",
-    "otherNight": 0,
+    "otherNight": 5,
```

Comment:
I don't think this makes sense. The Pixie is *not* woken to learn that they gained a new ability, per the KS almanac.

Reply:
Have checked with Steve and the Pixie showing up on other nights was an error in the upload of the nightsheet. This should be corrected shortly. @ratteler50 if you remove the Pixie other nights number, the rest of these can go in and the script tool will be updated soon to match.

Reply:
Send commited a change re-removing pixie from the nightsheet and shifting all other roles back down one.